acceptance/suites/pre_suite/puppet3_compat/70_install_puppet.rb

```diff
@@ -5,9 +5,15 @@
 legacy_agents.each do |host|
   platform = host.platform
 
-  puppet_version = ENV['PUPPET_LEGACY_VERSION']
-  if not puppet_version
+  if not ENV['PUPPET_LEGACY_VERSION']
     fail "PUPPET_LEGACY_VERSION is not set, e.g. '3.7.5'"
+  else
+    if platform =~ /(ubuntu|debian|cumulus)/
+      # Recommendation from QE in QENG-2385
+      puppet_version = "#{ENV['PUPPET_LEGACY_VERSION']}-1puppetlabs1"
+    else
```

Comment:
You could add a elseif or whatever ruby calls it to handle windoze using https://github.com/puppetlabs/beaker/blob/master/lib/beaker/dsl/install_utils/puppet_utils.rb#L347